### DISCUSSION TITLE
[feature] Due to special policies in China, you must use a VPN to connect to the wallet when using @web3modal/ethers "^4.1.5" and ethers "^6.11.1".

### DISCUSSION THREAD

**Discussion by liujiufa** [reown-com/appkit#3703] [Ideas]
### What problem does this new feature solve?

![Uploading photo_2024-09-19_18-54-09 (2).png…]()

### Describe the solution you'd like

I can connect with VPN, otherwise it keeps loading...

**Comment by liujiufa** [comment]
![photo_2024-09-19_18-54-09 (2)](https://github.com/user-attachments/assets/aef1c571-028e-41ae-8ff2-42ae5a6ea53a)

**Comment by rtomas** [comment]
cc @chris13524

**Comment by chris13524** [comment]
We do not have ability to investigate this as we do not have anyone based in China. @liujiufa you share some console errors or network errors regarding which request(s) are being blocked so we can resolve?
